src/index.html

```diff
@@ -3,11 +3,79 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="width=device-width, user-scalable=no, 
+          initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <ul class="stars stars--0">
+      <li class="stars__star"></li>
+
+      <li class="stars__star"></li>
+        
+      <li class="stars__star"></li>
+
+      <li class="stars__star"></li>
+
+      <li class="stars__star"></li>
+
+    </ul>
```

Comment:
Let's add empty line between logical blocks (not after each tag)
```suggestion
    <ul class="stars stars--0">
      <li class="stars__star"></li>
      <li class="stars__star"></li>
      <li class="stars__star"></li>
      <li class="stars__star"></li>
      <li class="stars__star"></li>
    </ul>
```